src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Visualize your network performance data in New Relic One
+tags:
+  - Integrations
+  - Network performance monitoring
+  - Monitoring
+metaDescription: Visualize network performance data in New Relic One.
+---
+
+After you've set up your network data for performance monitoring, you can go to New Relic One to install several custom curated visualizations designed specifically for network performance montoring.
+
+Once you've set up these visualizations, you'll be able to start exploring the network data your devices are reporting to New Relic One.
+
+## Prerequisites [#prerequisites]
+
+- Having set up your network performance monitoring devices.
+- Add the custom visualizations app to your account:
+    1. Go to [one.newrelic.com](https://www.one.newrelic.com) > **Apps**, and search for **Network Agent Visualizations**.
```

Comment:
I'd put one.newrelic.com in black too. BTW the links are not working in my local build but since I'm having so many issues I guess it's just me.

Reply:
It applies to all links in this doc!

Reply:
You're right! I fixed the links.